tests/bluefinClient.test.ts

```diff
@@ -45,7 +31,7 @@ Faucet.requestSUI(testAcctPubAddr, Networks.TESTNET_SUI.faucet);
 Faucet.requestSUI(testSubAccPubAddr, Networks.TESTNET_SUI.faucet);
 
 //* set environment from here
-const network = Networks.TESTNET_SUI;
+const network = Networks.LOCAL;
```

Comment:
Not sure if we want to change this - there's a GitHubAction check saying it doesn't exist